dist/memoranda1.0-rc3.1-20160212/src/net/sf/memoranda/util/localmessages/messages_be.properties

```diff
@@ -63,7 +63,6 @@ DEFAULT PROJECT = Standaard Project
 DELETE = Verwijderen
 DELETE PROJECT = Verwijder project
 DELETE RESOURCE TYPE = Verwijder bron type
-DELETE THIS STICKER? = Verwijder deze sticker?
```

Comment:
I made a new task for this in the Unassigned story, so we can take care of it later.